frontend/src/js/actions/workspaces/copyItem.ts

```diff
@@ -0,0 +1,46 @@
+import { copyItem as copyItemApi } from '../../services/WorkspaceApi';
+import {ThunkAction} from "redux-thunk";
+import {GiantState} from "../../types/redux/GiantState";
+import {AppAction, AppActionType, WorkspacesAction} from "../../types/redux/GiantActions";
+import {getWorkspace} from "./getWorkspace";
+
+export function copyItems(
+    workspaceId: string,
+    itemIds: string[],
+    newWorkspaceId?: string,
+    newParentId?: string
```

Comment:
Is there a call site where `newParentId` is passed? If not we could simplify by removing the parameter here and on the backend.

Reply:
maybe it's not necessary because we only copy to the root of the new workspace, whereas we can move to a different folder in the same workspace

Reply:
Ah yeah good spot! I was originally designing this with the intention that you would be able to copy to a folder within the same workspace, but when 'hold down ctrl' didn't work I gave up. I think it might be worth keeping this in though so if there is a use case in future the wiring is already there maybe?

Reply:
I've added a comment to explain this